src/layer/BingTiledImageLayer.js

```diff
@@ -47,19 +49,22 @@ define([
                 this.imageSize, this.imageSize);
 
             this.displayName = displayName;
-            this.pickEnabled = false;
+
+            // TODO: Picking is enabled as a temporary measure for screen credit hyperlinks to work (see Layer.render)
+            this.pickEnabled = true;
 
             this.detectBlankImages = true;
 
-            this.creditImage = WorldWind.configuration.baseUrl + "images/powered-by-bing.png"
+            this.creditImage = WorldWind.configuration.baseUrl + "images/powered-by-bing.png";
         };
 
         BingTiledImageLayer.prototype = Object.create(MercatorTiledImageLayer.prototype);
 
         BingTiledImageLayer.prototype.doRender = function (dc) {
             MercatorTiledImageLayer.prototype.doRender.call(this, dc);
             if (this.inCurrentFrame) {
-                dc.screenCreditController.addImageCredit(this.creditImage);
+                dc.screenCreditController.addStringCredit("Imagery courtesy of Bing maps", Color.DARK_GRAY, "https://www.bing.com/maps");
```

Comment:
Hyperlinks added to Bing layer for the purposes of having a functional test for this PR.